domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/Claim.java

```diff
@@ -19,15 +19,14 @@
 import java.util.List;
 import java.util.Optional;
 
-import static uk.gov.hmcts.cmc.domain.models.ClaimDocumentType.SEALED_CLAIM;
 import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
 
 // Create these fields in JSON when serialize Java object, ignore them when deserialize.
 @JsonIgnoreProperties(
     value = {"totalAmountTillToday", "totalAmountTillDateOfIssue",
         "amountWithInterestUntilIssueDate", "totalInterest",
         "serviceDate", "amountWithInterest", "directionsQuestionnaireDeadline",
-        "sealedClaimDocument"},
+        "claimDocument"},
```

Comment:
I think this needs to be ignored in both serialization and deserialization. It's no more simple getter now, it expects one argument to be passed. It will always be  `Optional.empty()` in JSON when serialize.

Reply:
Also, please verify if `sealedClaimDocument` URI is being used in any place i.e. front-end/s then serialize that URI.

Reply:
did a search and I don't see any refrences to the sealedClaimDocumeUri in the front end.